jest.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     // Replaces the following formats with an empty module.
     '^.+\\.(scss|css|svg|woff|woff2|mp4|webm)$': '<rootDir>/tests/emptyModule',
   },
+  reporters: ['<rootDir>/tests/fingers-crossed-reporter.js'],
```

Comment:
I think it's ready for a proper name! I'd suggest putting it in a directory called `tests/jest-reporters/`

Reply:
Isn't it ready for a proper name? We hope we won't need to output console logs, that's the idea behind this name :)